core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -627,8 +699,36 @@ public void setProperties(Map<String, String> properties) {
    * Changes posix file permission.
    *
    * @param path path of the file
-   * @param posixPerm standard posix permission like "777", "775", etc
+   * @param permission the permission to set in short format, e.g. 0777
+   * @throws IOException if a non-Alluxio error occurs
+   */
+  public abstract void setPermission(String path, short permission) throws IOException;
+
+  /**
+   * Gets the owner of the given path. An empty implementation should be provided if not supported.
+   *
+   * @param path path of the file
+   * @return the owner of the file
+   * @throws IOException if a non-Alluxio error occurs
+   */
+  public abstract String getOwner(String path) throws IOException;
```

Comment:
would it make sense to have provide getPermissionStatus and setPermissionStatus instead of the individual owner (we should be using either owner or user everywhere), group, and permission methods?

on a related note @apc999 :

I think that what we refer to in our code base as permission (e.g. 0700) is typically referred to mode (see for example man page for chmod). I think it would make more sense to use permission to describe the combination user, group, and mode together. So instead of PermissionStatus we would just have Permission, which contains user, group, and mode.